### ISSUE TITLE
Tag People in uploaded post

### ISSUE THREAD

**Issue by chandrakanth527** [LevPasha/Instagram-API-python#233]
It would be great if there is some way to do this. If you can give me the endpoints I might even try tweaking the code myself.
I am not able to reverse engineer http calls in my mobile

**Comment by LevPasha** [comment]
Guys, need your help!

**Comment by chandrakanth527** [comment]
@LevPasha  do you have sniffed messages for this activity?

**Comment by LevPasha** [comment]
@chandrakanth527 no. I developing only function that i need myself. Because of this i asking someone to help.

**Comment by spacerest** [comment]
Hi, submitted a pull request #523
